src/ntimporters/trello/importer.py

```diff
@@ -0,0 +1,344 @@
+"""Trello -> Nozbe Teams importer"""
+import json
+import random
+from typing import Optional, Tuple
+
+import openapi_client as nt
+from dateutil.parser import isoparse
+from ntimporters.trello.trello_api import TrelloClient
+from openapi_client import apis, models
+from openapi_client.exceptions import OpenApiException
+from openapi_client.model.id16_read_only import Id16ReadOnly
+from openapi_client.model.id16_read_only_nullable import Id16ReadOnlyNullable
+from openapi_client.model.timestamp_read_only import TimestampReadOnly
+from openapi_client.model_utils import ModelNormal
+
+
+class ImportException(Exception):
+    """Import exception"""
+
+
+def strip_readonly(model: ModelNormal):
```

Comment:
that's the hack to strip readonly fields before sending request to backend api, otherwise is not possible to create model by openapi generated client.

Reply:
ok, we'll resolve it later